Podcast/RecommendationsEndpointRequests.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  RecommendationsEndpointRequests.swift
+//  Podcast
+//
+//  Created by Jack Thompson on 8/27/18.
+//  Copyright © 2018 Cornell App Development. All rights reserved.
+//
+
+import Foundation
+import SwiftyJSON
+
+class FetchUserRecommendationsEndpointRequest: EndpointRequest {
+    var user: User
+    
+    init(user: User) {
+        self.user = user
+        
+        super.init()
+        
+        path = "/recommendations/users/\(user.id)/"
+        
+        httpMethod = .get
+    }
+    
+    override func processResponseJSON(_ json: JSON) {
+        var episodes: [Episode] = []
+        json["data"]["recommendations"].forEach{ _, episodeJson in
+            let episode = Cache.sharedInstance.update(episodeJson: episodeJson["episode"])
+            UserEpisodeData.shared.update(with: episodeJson["blurb"].string, for: user, and: episode)
+            episodes.append(episode)
+        }
+        processedResponseValue = episodes
+    }
+}
+
+class CreateRecommendationEndpointRequest: EndpointRequest {
+    
+    var episodeID: String
+    
+    init(episodeID: String, with blurb: String? = nil) {
```

Comment:
is blurb the right choice here? Can be a little confusing on what the blurb is

Reply:
Blurb is our app verbiage. A recast with a blurb is our way of "Retweeting a podcast with a comment"